api/v1/slurmcluster_types.go

```diff
@@ -103,10 +104,28 @@ type SlurmConfig struct {
 	// +kubebuilder:default="Cgroup,CPU_Bind,Gres,JobComp,Priority,Script,SelectType,Steps,TraceJobs"
 	// +kubebuilder:validation:Pattern="^((Accrue|Agent|AuditRPCs|Backfill|BackfillMap|BurstBuffer|Cgroup|ConMgr|CPU_Bind|CpuFrequency|Data|DBD_Agent|Dependency|Elasticsearch|Energy|Federation|FrontEnd|Gres|Hetjob|Gang|GLOB_SILENCE|JobAccountGather|JobComp|JobContainer|License|Network|NetworkRaw|NodeFeatures|NO_CONF_HASH|Power|Priority|Profile|Protocol|Reservation|Route|Script|SelectType|Steps|Switch|TLS|TraceJobs|Triggers)(,)?)+$"
 	DebugFlags string `json:"debugFlags,omitempty"`
+	// Identifies the type of task launch plugin (e.g. pinning tasks to specific processors)
+	//
+	// +kubebuilder:validation:Pattern="^((task/affinity|task/cgroup|task/none)(,)?)+$"
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="task/cgroup,task/affinity"
+	TaskPlugin string `json:"taskPlugin,omitempty"`
```

Comment:
I think we shouldn't allow changing this. The current implementation is strictly bound on task/cgroup plugin at the moment.

Reply:
https://github.com/nebius/soperator/pull/271/commits/673d5e8973a685f85be053ce6d8f9f4fc351ab06